Katello/katello
# Fixes #31654 - Support content migration reset

To test:

1) Switch over to Pulp 2 for yum, docker, and file
2) Sync content for each repo type
3) Run `bundle exec rake katello:pulp3_migration`
4) Check in the console for Pulp 3 DB entries such as:
  - Repo entries (version_href, remote_href, publication_href)
  - ::Katello::Pulp3::DistributionReference
  - ::Katello::Pulp3::RepositoryReference
  - Content unit entries (migrated_pulp3_href)
5) Run `bundle exec rake katello:pulp3_migration_reset
6) Check that the entries mentioned before are all `nil` or nonexistent
